src/main/resources/dev/definitions/prisons/orphanage/ors-property-location-history.json

```diff
@@ -20,8 +20,18 @@
-      "parameters": [],
+      "query": "dataset_base_ AS ( SELECT DISTINCT AT_OFFENDER.OFFENDER_ID_DISPLAY AS OFFENDER_ID_DISPLAY, AT_OFFENDER.FIRST_NAME AS FIRST_NAME, AT_OFFENDER.LAST_NAME AS LAST_NAME, DT_ALL_PROP_BOXES.SEAL_MARK AS SEAL_MARK, DT_ALL_PROP_BOXES.EXPIRY_DATE AS EXPIRY_DATE, AT_ALL_PROP_LOCS.DESCRIPTION AS DESCRIPTION, DT_ALL_PROP_BOXES.ACTIVE_FLAG AS ACTIVE_FLAG, DT_ALL_PROP_BOXES.CONTAINER_CODE AS CONTAINER_CODE, DT_ALL_PROP_BOXES.PROPERTY_CONTAINER_ID AS PROPERTY_CONTAINER_ID FROM OMS_OWNER.OFFENDERS AT_OFFENDER, OMS_OWNER.OFFENDER_BOOKINGS AT_OFFENDER_BOOKING, OMS_OWNER.AGENCY_INTERNAL_LOCATIONS AT_ALL_PROP_LOCS, ( SELECT Offender_Book_ID, Property_Container_ID, Active_Flag, Seal_Mark, Agy_Loc_ID, Expiry_Date, Internal_Location_ID, PROPOSED_DISPOSAL_DATE, Container_Code, Property_Only_Flag, Comment_Text FROM oms_owner.offender_ppty_containers WHERE Agy_Loc_ID = ( SELECT CASELOAD FROM context_ ) ) DT_ALL_PROP_BOXES WHERE ( AT_OFFENDER.OFFENDER_ID = AT_OFFENDER_BOOKING.OFFENDER_ID ) AND ( DT_ALL_PROP_BOXES.OFFENDER_BOOK_ID = AT_OFFENDER_BOOKING.OFFENDER_BOOK_ID ) AND ( AT_ALL_PROP_LOCS.INTERNAL_LOCATION_ID = DT_ALL_PROP_BOXES.INTERNAL_LOCATION_ID ) AND ( DT_ALL_PROP_BOXES.AGY_LOC_ID = ( SELECT ESTABLISHMENT_CODE FROM prompt_ /* = #Prompt('Establishment Code','A','Establishment\\\\\\\\Establishment Code',mono,free,,Not_Persistent,,User:3) */ ) ) ), dataset_ AS ( SELECT OFFENDER_ID_DISPLAY, FIRST_NAME, LAST_NAME, SEAL_MARK, EXPIRY_DATE, DESCRIPTION, ACTIVE_FLAG, CONTAINER_CODE, PROPERTY_CONTAINER_ID, CASE WHEN LENGTH(REGEXP_SUBSTR(SUBSTR(DESCRIPTION, 18), '\\d+')) > 0 THEN TO_NUMBER(REGEXP_SUBSTR(SUBSTR(DESCRIPTION, 18), '\\d+')) ELSE 0 END AS PROP_BOX_NO FROM dataset_base_ ) ",
```

Comment:
Why `Agy_Loc_ID = 'ALI'` was replaced with `Agy_Loc_ID = ( SELECT CASELOAD FROM context_ )`?
And then the prompt was put additionally at the end:
`DT_ALL_PROP_BOXES.AGY_LOC_ID = ( SELECT ESTABLISHMENT_CODE FROM prompt_`
I was expecting that `Agy_Loc_ID = 'ALI'` would have been replaced with the prompt. 
I am just trying to understand how this outcome was produced.